python/Ice/bidir/client/client.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) ZeroC, Inc.
+
+from EarlyRiser import AlarmClock, ButtonPressed
+
+class MockAlarmClock(AlarmClock):
```

Comment:
I would not call this file 'client.py'.

Reply:
What about `clock.py`?

Reply:
I would prefer mock.py or mock_alarm_clock.py.